tables/iRO/official/portals.txt

```diff
@@ -66,12 +66,14 @@ ama_dun01 229 7 ama_in02 119 181 0 c r0 n
 #prontera 282 200 morocc 156 47 1200 c r2 c r3
 #prontera 282 200 gef_fild10 52 326 1700 c r2 c r4
 #prontera 282 200 alberta 117 56 1800 c r2 c r5
-#prontera 29 207 izlude 94 103 600 c r2 c r0
-#prontera 29 207 geffen 120 39 1200 c r2 c r1
-#prontera 29 207 payon 161 58 1200 c r2 c r2
-#prontera 29 207 morocc 156 47 1200 c r2 c r3
-#prontera 29 207 gef_fild10 52 326 1700 c r2 c r4
-#prontera 29 207 alberta 117 56 1800 c r2 c r5
+prontera 35 208 izlude 94 103 2000 1 c r2 c r0
+prontera 35 208 geffen 120 39 2000 1 c r2 c r1
+prontera 35 208 payon 161 58 2000 1 c r2 c r2
+prontera 35 208 morocc 156 47 2000 1 c r2 c r3
+prontera 35 208 alberta 117 56 2000 1 c r2 c r4
+prontera 35 208 aldebaran 168 112 2000 1 c r2 c r5
+prontera 35 208 comodo 209 143 3000 1 c r2 c r6
+prontera 35 208 umbala 100 154 3000 1 c r2 c r7
```

Comment:
These are under VIP Warps.
Should replace under:
https://github.com/OpenKore/openkore/blob/0842ac7990a41aecc7b91130de33afea7ee59d0c/tables/iRO/official/portals.txt#L95
There will be a conflict later.

And if wanna change, should change VIP too.
But I prefer adding it as Optional rather than default.

Reply:
roger that sir

i wonder why the vip warps of this specific npc are exactly equal to the non-VIP warps

i'm adressing these issues now

Reply:
VIP is IRO is like "Unlock" feature from normal stuff.
If you have VIP, Kafra just offer 2nd storage, more destination, less fee and etc etc.